README.md

```diff
@@ -236,98 +220,72 @@ brew update; brew install kubernetes-helm
 
 ```bash
 helm init
+kubectl create clusterrolebinding tiller-clusterrole-binding --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
 ```
 
+*Note:* This setup will deploy the helm server component and will give helm access to the whole cluster. For a more secure, customized setup, please read -> https://helm.sh/docs/using_helm/#role-based-access-control
+
 ### 7. Add the Alfresco Incubator Helm Repository
 
 ```bash
 helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
 ```
 
 ### 8. Add Local DNS
 
-Add Local DNS Entry for Host Machine (needed for JWT issuer matching). Be sure to specify an active network interface.  It is not always `en0` as illustrated.  You can use the command `ifconfig -a` to find an active interface.
+We will be forming a local dns with the use of nip.io. All you have to do it get your ip using the following command.
 
 ```bash
-sudo sh -c 'echo "`ipconfig getifaddr en0`       localhost-k8s" >> /etc/hosts'; cat /etc/hosts
+ipconfig getifaddr en0
```

Comment:
As I think Jared pointed out previously, it may not always be `en0`, but hopefully users installing a DBP understand the basics of network interfaces.

Reply:
We hope so.